src/screens/NewBook/index.jsx

```diff
@@ -0,0 +1,72 @@
+import Navbar from '@/components/Navbar'
+import { MainContainer, FullyContainer } from '@/styles/Containers'
+
+import Image from 'next/image'
+import ImgCapa from '@/assets/img/img-capa.png'
+
+import * as S from './styles'
+
+const NewBook = () => {
```

Comment:
Pra evitar ter o nome da página e o da screen igual, a gente pode fazer igual fiz na `Home`. Ao componente screen da `home`, dei o nome `HomeScreen`

```suggestion
const NewBookScreen = () => {
```

A gente teria algo assim ☝️ nesse caso.